### ISSUE TITLE
Can I run 2 vibration sensors?

### ISSUE THREAD

**Issue by redfive2012** [Shmoopty/rpi-appliance-monitor#3]
Is it possible to connect and configure 2 vibration sensors on the same RPi? I'd like to put one each on my washer and dryer, and I have a 1st-gen B+ model that would be great if it could run 2 sensors.

**Comment by Shmoopty** [comment]
Yes, a single RPi can monitor many sensors.  I can confirm that first-hand.

Take a look at the [sample settings file](https://raw.githubusercontent.com/Shmoopty/rpi-appliance-monitor/master/vibration.py) and note the setting `sensor_pin`.  In the photos, you can see the one sensor connecting to pin 14.  Foillowing the normal instructions, here are your extra steps:

* Attach a second vibration module to the *same* 5V and GND pins, but a *different* sensor GPIO pin.  You'll want to use a very flexible or long  cable, so one vibrating sensor doesn't vibrate everything.
* Create two settings files with their own timings, messages, and unique sensor pins.
* Add two lines to `/etc/rc.local`.  One for each settings file.

Good luck!

**Comment by redfive2012** [comment]
Nice! Thanks. This would be a great addition to my network.

**Comment by sorford638** [comment]
Hi,

You gae me some great advice last month and my parts finally arrived today so I've been tinkering all evening.  However, I can only get one sensor working.  I've checked the components by swapping the wires and sensors around but I can only get the dryer sensor to work.  It seems like no coincidence that this is the one that you gave the detailed instructions for!

In my setup I've got three sensors and I've made three copys of the python script (is this necessary?) and three settings files to go with them.  I've then added the details to rc.local using the same format as you.  I've attached photos to show where I'm at.

I'm sure that my mistake is glaringly obvious but I just can't see it!

Thanks again for your help,
Steve

![rc-local](https://cloud.githubusercontent.com/assets/10810507/23817041/8f43bf52-05f0-11e7-8c51-1c064ff0c0b5.png)
![files](https://cloud.githubusercontent.com/assets/10810507/23817054/9f450618-05f0-11e7-8cd7-73c8028cc47c.png)
